tests/main/component-from-store/task.yaml

```diff
@@ -0,0 +1,28 @@
+summary: Test installing a component from the store
+
+details: |
+  Verifies that we can install a snap and components from the store at the same
+  time.
+
+systems: [ubuntu-16.04-64, ubuntu-18.04-64, ubuntu-2*, ubuntu-core-*, fedora-*]
+
+restore: |
+  snap remove test-snap-with-components || true
+
+  # TODO:COMPS: remove this once removing a snap also removes components
+  systemctl stop -- '*test\x2dsnap\x2dwith\x2dcomponents-components-mnt-*.mount' || true
+  rm /etc/systemd/system/*'-test\x2dsnap\x2dwith\x2dcomponents-components-mnt-*.mount' || true
```

Comment:
you have a literal `*` between single quotes here

Reply:
I think we want that/is ok, right? Since the glob gets handled by systemctl, not the shell.

Reply:
I am talking about the one in the `rm` command

Reply:
Ah thank you